apps/camera_pipe/camera_pipe.cpp

```diff
@@ -306,6 +307,7 @@ Func process(Func raw, Type result_type,
         .split(x, x, xi, 2*vec, TailStrategy::RoundUp)
         .reorder(xi, c, yii, x, yi, yo)
         .vectorize(xi, 2*vec)
+        .prefetch(yo, 2)
```

Comment:
Yes.  This can be removed.  I originally had a problem with prefetching on a loop variable introduced from a split so I kept it around to make sure it stayed fixed as things changed.